exploration-module/src/main/java/io/edpn/backend/exploration/adapter/persistence/MybatisSystemEliteIdRequestRepository.java

```diff
@@ -21,7 +21,7 @@ public interface MybatisSystemEliteIdRequestRepository {
             "ON CONFLICT (requesting_module, system_name)",
             "DO NOTHING"
     })
-    void insert(MybatisSystemEliteIdRequestEntity requestDataMessageEntity);
+    void insertIfNoExists(MybatisSystemEliteIdRequestEntity requestDataMessageEntity);
```

Comment:
There's a subtle difference here: "insertIfNotExist" and "insertIfNoExists". And IMHO they both miss the mark: in DB syntax it's typically "IF NOT EXISTS", so I guess the method should be "insertIfNotExists"?

Reply:
typo fixed